manifests/rule.pp

```diff
@@ -13,6 +13,16 @@
 #      table   => 'ip6-nat'
 #  }
 #
+# @example Redirect port 443 to port 8443
+#  nftables::rule{'PREROUTING-redirect':
```

Comment:
Style is not consistent with the previous examples, but none match what is recommended by [the puppet style guide](https://www.puppet.com/docs/puppet/8/style_guide.html#style_guide_module_design-spacing). Should be:

```suggestion
#  nftables::rule { 'PREROUTING-redirect':
```